index.html

```diff
@@ -2225,25 +2032,31 @@ <h4 id="optional-features">Success Criterion: Optional features</h4>
 				<section class="notoc" data-testable="machine" data-materials="medium" data-energy="medium" data-water="medium" data-emissions="medium" data-standard="" data-considerations="Accessibility" data-categories="Compatibility, Content, HTML, Social Equity, Usability">
 					<h4 id="non-standard-code">Success Criterion: Non-standard code</h4>
 					<p class="external"><a class="machine" href="https://w3c.github.io/sustainableweb-wsg/star.html#WD07-3">Machine-testable</a> <span class="hide">and </span><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#WD07-3">Resources</a></p>
-					<p>Avoid using non-standard HTML elements or attributes.</p>
+					<p>Avoid using non-standard, deprecated, proprietary, or outdated formats and web standards. Only use such code where this is required to meet a documented customer need and if there is a justifiable benefit that cannot otherwise be met. Justifiable reasons could include compatibility with essential legacy systems and/or hardware, accessibility, or emissions reduction. Use polyfills only when necessary, and regularly audit code to see if they can be removed.</p>
 				</section>
 				<section class="notoc" data-testable="human" data-materials="medium" data-energy="medium" data-water="medium" data-emissions="medium" data-standard="GPF" data-considerations="Accessibility" data-categories="Compatibility, Content, HTML, Social Equity, Usability">
 					<h4 id="custom-code">Success Criterion: Custom code</h4>
 					<p class="external"><span class="human">Human-testable</span> <span class="hide">and </span> <a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#WD07-4">Resources</a></p>
 					<p>Prioritize the use of standard HTML elements and attributes. Only use custom elements or Web Components if you cannot use native elements or if you require them for the purposes of producing reusable design system components.</p>
 				</section>
+				<section class="notoc" data-testable="machine" data-materials="medium" data-energy="medium" data-water="medium" data-emissions="medium" data-standard="GPF" data-considerations="" data-categories="Compatibility, CSS, HTML, JavaScript, Patterns, UI">
+					<h4 id="native-over-custom">Success Criterion: Native over custom</h4>
+					<p class="external"><a class="machine" href="https://w3c.github.io/sustainableweb-wsg/star.html#WD07-5">Machine-testable</a> <span class="hide">and </span><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#WD07-5">Resources</a></p>
+					<p>Use native functions, APIs, and features over writing your own.</p>
```

Comment:
native often refers to proprietary (as in iOS or android. Do we mean Web? built-in?